ArduPlane/quadplane.cpp

```diff
@@ -3533,7 +3539,7 @@ float QuadPlane::forward_throttle_pct()
             return 0;
         } else {
             // calculate fwd throttle demand from manual input
-            float fwd_thr = rc_fwd_thr_ch->percent_input();
+            float fwd_thr = rc_fwd_thr_ch->norm_input()*100.0f;
 
             // set forward throttle to fwd_thr_max * (manual input + mix): range [0,100]
             fwd_thr *= .01f * constrain_float(fwd_thr_max, 0, 100);
```

Comment:
This means you get symmetrical +- `fwd_thr_max` in the new case. Not `fwd_thr_min` to `fwd_thr_max`.

Some thing like:
```
if (!is_negative(fwd_thr )) {
    fwd_thr *= 0.01 * constrain_float(fwd_thr_max, 0, 100);
} else {
    fwd_thr *= 0.01 * constrain_float(-fwd_thr_min, 0, 100);
}
```
Probably also makes more sense to do the percent to ration on the `percent_input` call rather than multiplying up on the `norm_input` and then back down again here.